src/app/store/reducers.ts

```diff
@@ -24,6 +25,7 @@ const reducer: Reducer<RootState> = (state, action) => {
     manifest,
     vendors,
     compare,
+    streamDeck,
```

Comment:
This should be guarded by a feature flag as well. Ideally the whole StreamDeck feature loads async, and doesn't load unless activated.

Reply:
ok, I'll check it later